### PR TITLE
rMVP

### PR DESCRIPTION
**Pull request by collin**
@spra85  We have the beginnings of a rail player.

### CODE REVIEW

File: elements/bulbs-video/components/revealed.js [hunk 1]
```diff
@@ -206,6 +206,7 @@ export default class Revealed extends React.Component {
       flashplayer: '//ssl.p.jwpcdn.com/player/v/7.4.3/jwplayer.flash.swf',
       aspectratio: '16:9',
       autostart: true,
+      hlshtml: true,
```
Comment (collin) [inline elements/bulbs-video/components/revealed.js]: @spra85 What's the story on this setting?

**Comment by collin** [comment]
@spra85 @daytonn @kand 

`scripts/webpack-dev-server` and:

http://localhost:8080/example/rail-player/with-channel-logo

**Comment by kand** [comment]
👍 looks great

File: elements/bulbs-video/components/revealed.js (hunk 1 above)
Comment (spra85) [inline elements/bulbs-video/components/revealed.js]: @collin, we have to omit that until we are HTML5 certified by an ad tech vendor for one of our live campaigns.  JWPlayer should have a bug fix for that one issue in their 7.5 beta release that's ready this week or next.

File: elements/bulbs-video/elements/rail-player/components/root.js [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+            className='rail-player-recirc-link'
+            target='_blank'
+            href={this.props.recircUrl}
+          >
+            Watch More
+          </a>
+        </div>
+
+        <div className='rail-player-video bulbs-video-root player'>
+          <Revealed disableSharing={true} {...this.props}/>
```
Comment (spra85) [inline elements/bulbs-video/elements/rail-player/components/root.js]: @collin, we'll need to pass in the muted property so this is true: https://github.com/theonion/bulbs-elements/blob/master/elements/bulbs-video/components/revealed.js#L105

File: elements/bulbs-video/elements/rail-player/rail-player.js [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+    if (this.props.src !== prevProps.src) {
+      this.initialDispatch();
+    }
+  }
+
+  render () {
+    return (
+      <RailPlayerRoot
+        {...this.state}
+        muted={typeof this.props.muted === 'string'}
```
Comment (spra85) [inline elements/bulbs-video/elements/rail-player/rail-player.js]: Re: my other comment, not sure if we even want the muted part to be optional...should always be muted by default

**Comment by spra85** [comment]
👍 other than the comments about muting.  Will need a style review from the design team

**Comment by spra85** [comment]
<a href="https:&#x2F;&#x2F;trello.com&#x2F;c&#x2F;FcEgWVU3&#x2F;21-bulbs-elements-custom-rmvp-component"> Bulbs Elements - Custom rMVP component</a>

**Pull request merged**